pygfx/utils/transform.py

```diff
@@ -658,3 +661,7 @@ def __matmul__(self, other):
             return RecursiveTransform(other, parent=self)
         else:
             return np.asarray(self) @ other
+
+    @cached
+    def _decomposed(self) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+        return la.mat_decompose(self.matrix)
```

Comment:
Scale factors not not needed here?

Reply:
This is on Recursive transform, a subclass of Affine transform. We use this for world transforms.